Covid update 3 - New Portland conference dates (#1210)
writethedocs/www: docs/conf/portland/2020/news/covid-19-update-03.rst

:template: {{year}}/generic.html

.. post:: March 18, 2020
   :tags: portland-2020, covid-19

COVID-19 Update 03 - New Portland Conference Dates
==================================================

Hey Folks,

> We hope that you're doing okay. It's been a stressful few weeks, and we wanted to send positive thoughts as we all attempt to get through this. `Please let us know`_ if there is anything we can do to help you. 

Reviewer: <3

:template: {{year}}/generic.html

.. post:: March 18, 2020
   :tags: portland-2020, covid-19

COVID-19 Update 03 - New Portland Conference Dates
==================================================

Hey Folks,

> We hope that you're doing okay. It's been a stressful couple weeks, and we wanted to send positive thoughts as we all attempt to get through this. `Please let us know`_ if there is anything we can do to help you. 

Reviewer 2: ```suggestion
We hope that you're doing okay. It's been a stressful few weeks, and we wanted to send positive thoughts as we all attempt to get through this. `Please let us know`_ if there is anything we can do to help you. 
```

The true power of community comes in the most trying of times. It's been amazing watching folks come together on :doc:`Slack </slack>`, and we're trying to plan more ways for folks to feel connected and supported.

We also wanted to give you an update on the Portland conference. **We have a new date, August 9-11, 2020.**

Given all the uncertainty about the future, we are being cautious in our approach with the conference.
We're announcing the new date so that people can mark it on their calendars,

> but we're opening ticket sales again until we are certain that the event can proceed. 

Reviewer 2: ```suggestion
but we're waiting to open ticket sales again until we are certain that the event can proceed. 
```

> We know that having a firm dates means that not everyone will be able to come.

Reviewer 2: ```suggestion
We know that changing the conference dates means that not everyone will be able to attend.
```

> In order to more easily handle requests for changes, we've `created a form`_ to handle incoming ticket change requests.

Reviewer 2: ```suggestion
In order to more easily handle requests for changes, we `created a form`_ to handle incoming ticket change requests. 
```
